libgit/browser.go

```diff
@@ -197,7 +203,11 @@ func (b *Browser) Lstat(filename string) (fi os.FileInfo, err error) {
 
 	// Git doesn't keep track of the mtime of individual files
 	// anywhere, so just use the timestamp from the commit.
-	return &browserFileInfo{entry, size, b.mtime}, nil
+	fi = &browserFileInfo{entry, size, b.mtime}
+
+	b.sharedCache.setFileInfo(b.commitHash, path.Join(b.root, filename), fi)
```

Comment:
I don't see a corresponding `getFileInfo` here or in `Stat()`.

Reply:
good catch!